### ISSUE TITLE
Error during building custom image

### ISSUE THREAD

**Issue by hypebeast** [vollstock/Tooloop-OS#2]
I tried to build a custom image (following the guide at https://github.com/vollstock/Tooloop-OS/wiki/building-an-iso) and I get the following error message:

```
Reading in package information for amd64:
  Done: Read details of 716 packages for amd64
WARNING: no translated descriptions found for jessie/main
Starting to lay out packages into images:
  Adding the required directories
  Generating the image label and volume id
  Adding .disk/base_components
  Adding .disk/cd_type
  Adding udeb/base includes/excludes
WARNING: Unable to read UDEB_EXCLUDE file /home/vagrant/Development/tooloop-os/tmp//debian-cd/data/jessie/amd64_netinst_udeb_exclude
  Adding docs to CD1
  Extracting FAQ on CD1
mkdir: cannot create directory '/home/vagrant/Development/tooloop-os/tmp//cd-build/jessie/CD1/doc/FAQ/html': No such file or directory
    Failed to start disc 1, error 256
Makefile:408: recipe for target 'image-trees' failed
make: *** [image-trees] Error 1
including description: /home/vagrant/Development/tooloop-os/profiles/tooloop.description
simple-cdd: extra files for simple-cdd
find: `/home/vagrant/Development/tooloop-os/tmp//cd-build/jessie/CD1/pool': No such file or directory
ERROR: missing required packages from profile default:  less
ERROR: missing required packages from profile default:  simple-cdd-profiles
WARNING: missing optional packages from profile default:  grub-pc grub-efi popularity-contest console-tools console-setup usbutils acpi acpid eject lvm2 mdadm cryptsetup reiserfsprogs jfsutils xfsprogs debootstrap busybox syslinux-common syslinux isolinux
ERROR: missing required packages from profile tooloop:  xorg x11-xserver-utils openbox obmenu obconf compton ssh x11vnc pulseaudio unclutter git sudo unzip make gcc augeas-doc augeas-lenses augeas-tools
```

**Comment by vollstock** [comment]
Found it :-) I set up a new virtual build machine and ran into the same problem.

I didn't quite accurately understand/read/copy the noted from the [simple-cdd How-Tos](https://wiki.debian.org/Simple-CDD/Howto).

Apparently it's mandatory to first build a CD without a profile so the local mirror is created.
Do this by first run `build-simple-cdd`without a profile,
It’s still a good idea to first find a local mirror that’s faster then one across the world.

E.g. this should build a standard CD:

    build-simple-cdd --debian-mirror "http://ftp.de.debian.org/debian/"

After having done that, the build *with* profile worked just fine:

    build-simple-cdd --profiles tooloop

I have updated the wiki accordingly :-)
Make sure to check out the latest git, though. I have changed some things.
